CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Localize placeholder based on language set in constructor options [#150](https://github.com/mapbox/mapbox-gl-geocoder/issues/150)
 - `trackProximity` turned on by default [#195](https://github.com/mapbox/mapbox-gl-geocoder/issues/195)
 - Bump suggestions to v1.3.4
+- Add constructor properties that allow adding the selected result to the map as a mapbox-gl#marker. Adding the result to the map is now the default behavior. 
```

Comment:
Something I've personally found helpful in CHANGELOG's is separating out breaking changes, features, and bug fixes, eg https://github.com/mapbox/mapbox-gl-js/blob/master/CHANGELOG.md#0490. This makes it easier to see the most important changes like breaking changes.

Reply:
This is a great example and one we can emulate going forward.